worker/buildbot_worker/commands/fs.py

```diff
@@ -161,7 +161,7 @@ def start(self):
         fromdir = os.path.join(self.builder.basedir, self.args['fromdir'])
         todir = os.path.join(self.builder.basedir, self.args['todir'])
 
-        self.timeout = args.get('timeout', 120)
+        self.timeout = args.get('timeout', None)
```

Comment:
I don't think this is the right fix.
```
>>> args = {'timeout': None}
>>> print(args.get('timeout', 120))
None
```
args.get should be returning None is args['timeout'] is None.

aren't we rather removing None argument before?